data/json/npcs/exodii/exodii_merchant_talk.json

```diff
@@ -383,6 +419,215 @@
+          {
+            "u_spawn_npc": "BEM_prototype_cyborg_with_yonked_bionics",
+            "real_count": 1,
+            "outdoor_only": false,
+            "min_radius": 1,
+            "max_radius": 3,
+            "spawn_message": "Rubic leads a person to you."
+          }
+        ],
+        "topic": "TALK_DONE"
+      },
+      {
+        "text": "Let's see what I get.",
+        "condition": { "u_has_var": "dialogue_exodii_cybor_rescue_ongoing_current_type", "value": "pull_out_from_rotten_cyborg" },
+        "effect": [
+          { "u_add_var": "dialogue_exodii_cybor_rescue_ongoing", "value": "no" },
+          { "u_add_var": "dialogue_exodii_cybor_rescue_ongoing_current_type", "value": "none" },
+          { "u_spawn_item": "ruined_chunks", "count": 20 },
+          { "u_spawn_item": "bionics_pulled_out_prototype", "use_item_group": true }
+        ],
+        "topic": "TALK_DONE"
+      },
+      {
+        "text": "It is time to put this body to rest.",
+        "condition": { "u_has_var": "dialogue_exodii_cybor_rescue_ongoing_current_type", "value": "dress_up_rotten_cyborg" },
+        "effect": [
+          { "u_add_var": "dialogue_exodii_cybor_rescue_ongoing", "value": "no" },
+          { "u_add_var": "dialogue_exodii_cybor_rescue_ongoing_current_type", "value": "none" },
+          { "u_spawn_item": "corpse_generic_human" }
+        ],
+        "topic": "TALK_DONE"
+      },
+      { "text": "Never mind, I want to ask you something.", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "Actually, forget it.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Experiment_Cyborg_Rescue",
+    "type": "talk_topic",
+    "dynamic_line": { "//~": "Let me see.  Will do what I can for a coin.", "str": "Lemme peep.  Will do what I can for a bob." },
+    "responses": [
+      {
+        "text": "[Prototype Cyborg] Can you help this guy look like they have lost their mind?",
+        "condition": { "u_has_items": { "item": "bot_prototype_cyborg", "count": 1 } },
+        "topic": "TALK_EXODII_MERCHANT_Experiment_Prototype_Cyborg_Rescue"
+      },
+      {
+        "text": "[Broken Cyborg] Can you save my buddy who look a bit rough around the edges, but he still didn't flatline?",
+        "condition": { "u_has_items": { "item": "bot_broken_cyborg", "count": 1 } },
+        "topic": "TALK_EXODII_MERCHANT_Experiment_Broken_Cyborg_Rescue"
+      },
+      { "text": "Actually, forget it's not my problem.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Experiment_Prototype_Cyborg_Rescue",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "//~": "For this one I can pull out their cbm and give workingt to you, or I can just take out broken bits so that they will still be able to use the rest of stuff in them.  Alternatively why don't you just give him to me i surelly can tun him in quite usefull worker.'",
```

Comment:
```suggestion
      "//~": "For this one I can pull out their cbm and give any working to you, or I can just take out broken bits so that they will still be able to use the rest of stuff in them.  Alternatively why don't you just give him to me I surely can turn him into quite the useful worker.'",
```